docs/how to/use-transferbench.rst

```diff
@@ -1,106 +1,130 @@
-This simplifies to a simple copy operation when dealing with single SRC/DST.
+A transfer is defined as a single operation where an executor reads and adds together values from SRC memory locations, then writes the sum to the DST memory locations.
+This simplifies to a simple copy operation when dealing with a single SRC or DST.
+Here is a copy operation from a single SRC to DST:
 
 .. code-block:: bash
 
    SRC 0                DST 0
    SRC 1 -> Executor -> DST 1
    SRC X                DST Y
 
-Three Executors are supported by TransferBench::
+Three Executors are supported by TransferBench:
 
-   Executor:        SubExecutor:
-   1) CPU           CPU thread
-   2) GPU           GPU threadblock/Compute Unit (CU)
-   3) DMA           N/A.                                 (May only be used for copies (single SRC/DST)
+Executor:        SubExecutor:
+1. CPU           CPU thread
+2. GPU           GPU threadblock/Compute Unit (CU)
+3. DMA           N/A (Can only be used for a single SRC to DST copy)
 
-Each single line in the configuration file defines a set of Transfers (a Test) to run in parallel
+Each line in the configuration file defines a set of transfers, also known as a test, to run in parallel.
 
 There are two ways to specify a test:
 
-1) Basic
+- Basic
 
-   The basic specification assumes the same number of SubExecutors (SE) used per Transfer
-   A positive number of Transfers is specified followed by that number of triplets describing each Transfer
+   The basic specification assumes the same number of SEs used per transfer.
+   A positive number of transfers is specified, followed by the number of SEs and triplets describing each transfer:
 
    Transfers SEs (srcMem1->Executor1->dstMem1) ... (srcMemL->ExecutorL->dstMemL)
 
-2) Advanced
+   .. code-block:: bash
+
+      1 4 (G0->G0->G1)                   Uses 4 CUs on GPU0 to copy from GPU0 to GPU1
+      1 4 (G2->C1->G0)                   Uses 4 CUs on GPU2 to copy from CPU1 to GPU0
+      2 4 G0->G0->G1 G1->G1->G0          Copies from GPU0 to GPU1, and GPU1 to GPU0, each with 4 SEs
+
+- Advanced
+
+   A negative number of transfers is specified, followed by quintuplets describing each transfer.
+   A non-zero number of bytes specified will override any provided value.
+
+   Transfers (srcMem1->Executor1->dstMem1 SEs1 Bytes1) ... (srcMemL->ExecutorL->dstMemL SEsL BytesL)
+
+   .. code-block:: bash
 
-   A negative number of Transfers is specified, followed by quintuplets describing each Transfer
-   A non-zero number of bytes specified will override any provided value
+      -2 (G0 G0 G1 4 1M) (G1 G1 G0 2 2M) Copies 1Mb from GPU0 to GPU1 with 4 SEs and 2Mb from GPU1 to GPU0 with 2 SEs
 
-   -Transfers (srcMem1->Executor1->dstMem1 SEs1 Bytes1) ... (srcMemL->ExecutorL->dstMemL SEsL BytesL)
+Here is the list of arguments:
```

Comment:
@gilbertlee-amd Where are these arguments specified? In config file?

Reply:
https://github.com/ROCm/TransferBench/blob/a7315688c50d84af9cf1f9f100bc0b12d51f0b4c/examples/example.cfg#L32

Yes